runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -358,10 +360,35 @@ public void hideWidgets(WidgetHiddenChanged event)
 		{
 			return;
 		}
+		fileName = fileName + "";
```

Comment:
?

Reply:
Kinda looks like a null safeguard for the filename string, but it's not necessary with the null check right above it.